en/advanced_config/esc_calibration.md

```diff
@@ -49,6 +49,6 @@ To calibrate the ESCs:
 
 :::note
 High-quality controllers come with a factory calibration.
-In *theory* this means that they can be configured by just setting the [PWM_MAIN_MINn](../advanced_config/parameter_reference.md#PWM_MAIN_MIN1)/[PWM_AUX_MINn](../advanced_config/parameter_reference.md#PWM_AUX_MIN1) and [PWM_MAIN_MAXn](../advanced_config/parameter_reference.md#PWM_MAIN_MAX1)/[PWM_AUX_MAXn](../advanced_config/parameter_reference.md#PWM_AUX_MAX1) parameters to the values provided in the ESC technical specification.
-In practice the input range may differ even on high quality controllers, which is why calibration is recommended. 
+In *theory* this means that the outputs might be can be configured by setting the values provided in the ESC technical specification for each output during [Actuator Configuration](../config/actuators.md) (under the hood this sets the [PWM_MAIN_MINn](../advanced_config/parameter_reference.md#PWM_MAIN_MIN1)/[PWM_AUX_MINn](../advanced_config/parameter_reference.md#PWM_AUX_MIN1) and [PWM_MAIN_MAXn](../advanced_config/parameter_reference.md#PWM_MAIN_MAX1)/[PWM_AUX_MAXn](../advanced_config/parameter_reference.md#PWM_AUX_MAX1) parameters).

```

Comment:
@bkueng Is this still required/supported? I mean the actuator config suggest moving the sliders to the first stop point and then modifying the PWM min until motor just moves. This is effectively manual motor calibration right?

If I'm right, what do we "recommend"?

Further, my understanding is you only need this for some ESC - PWM variants, including OneShot right? Not Dshot or UAVCAN?

Reply:
> This is effectively manual motor calibration right?

Right, but it's also a bit imprecise, and I'd still recommend calibration so all ESCs behave the same.

> Further, my understanding is you only need this for some ESC - PWM variants, including OneShot right? Not Dshot or UAVCAN?

Correct.